article/_posts/2021-07-09-SigmaVAE.md

```diff
@@ -0,0 +1,168 @@
+    title: "Simple and Effective VAE Training with Calibrated Decoders"
+    venue: "arXiv (rejected at ICLR 2021; accepted at ICML 2021)"
+pdf: "https://arxiv.org/pdf/2006.13202.pdf"
+---
+
+
+# Highlights
+
+- Their contributions are:
+ - A careful choice of the decoder distribution on naïve calibrated decoders
+often lead to worse results, can work very well, and removes the need to tune
+the additional parameter $$#beta$$.
+ - Novel technique, dubbed as $$sigma$$-VAE, for optimizing the decoder variance
+analytically, without requiring the decoder network to produce it as an
+additional output.
+- their results improve sample quality on various datasets compared to different
+VAE and sequence VAE models.
+
+
+# Introduction
+
+VAEs require caareful manual tuning of the balance between two terms that
+correspond to distortion (data reconstruction) and rate from information theory
+(KL divergence): a model with low rate would not contain enough information to
+reconstruct the data, while allowing the model to have high rate might lead to
+unrealistic samples from the prior as the KL-divergence constraint becomes
+weaker.
+
+$$\beta$$-VAEs allow to weigh the trade-off through the $$\beta$$ weighting
+term. However, the parameter needs to be tuned for every dataset and model
+variant as a hyperparameter.
+
+The paper revisits the idea of learning calibrated decoders. Most VAE models are
+employed with uncalibrated decoding distributions, such as Gaussian
+distributions without a learned variance, where the decoder only outputs the
+mean parameter.
+
+The work proposes to a VAE with calibrated uncertainties for the likelihood,
+which is say VAEs in which the variance is learned rather than chosen as a
+fixed hyperparameter. 
+
+
+# Calibrated decoders
+
+Due to various reasons (numerical instability, poor results of naïve
+approaches), calibrated decoders have not gained larger adoption by
+practitioners.
+
+Gaussian decoders with the common MSE reconstruction loss assume a particular,
+constat variance of the Gaussian distribution, which is a limitation. 
+
+The calibration of a neural network can be usually improved by estimating the
+uncertainty of that prediction, such as the variance of a Gaussian. Since the
+naïve MSE loss assumes a constant variance, it does not effectively represent
+the uncertainty of the prediction. Thus common VAEs using the MSE reconstruction
+loss are poorly calibrated.
+
+
+Authors propose to learn the variance (i.e. calibrate the decoder) a better
+performance might be achieved. 
+
+
+# Methods
+
+Learning the variance can be done by letting a neural network
+$$\sigma_{\theta}$$ output the diagonal entries of the covariance matrix given a
+latent sample $$z$$:
+
+![](/article/images/sigmaVAE/Naive_likelihood.jpg)
+
+A simpler and more stable parameterization is to specify the covariance
+matrix as a diagonal matrix with single **shared** parameter across pixels
```

Comment:
Punctuation missing at the end?

Reply:
Not really, the math expression on the next line follows the sentence.